### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by hucste**
- OpenBSD versions
- others informations

### CODE REVIEW

File: content/fact/chrooted-webserver.md [hunk 1]
```diff
@@ -9,7 +9,21 @@ webserver installation running.
...
+class
+gecos   HTTP Server
+dir     /var/www
+shell   /sbin/nologin
+expire  NEVER
+```
+
+```
+$ grep "www" /etc/passwd
 www:*:67:67:HTTP Server:/var/www:/sbin/nologin
```
Comment (noqqe) [inline content/fact/chrooted-webserver.md]: I find this too long and complicated to demonstrate what its for. 

Maybe only the `grep` part?

File: content/fact/opensmtpd.md [hunk 2]
```diff
@@ -10,6 +10,9 @@ daemon was needed for the base system of the operating system and no there
 were no suitable alternatives. The goals are simplicity, security and
 reliability with an acceptable [license](/fact/license/) (ISC).
 
+A major change in configuration was made with version 6.4, integrated into
+ OpenBSD 6.4.
+
 ```
```
Comment (noqqe) [inline content/fact/opensmtpd.md]: You are right, but why is this important? 

Would remove it and only focus on the tiny keypoints.

Users interested in OpenBSD should not be bothered with things were in the past.

File: content/fact/chrooted-webserver.md (hunk 1 above)
Comment (hucste) [reply content/fact/chrooted-webserver.md]: yes, you can only use `grep`....
but, `userinfo` is more adequate. 

It's up to you! ;)

File: content/fact/opensmtpd.md (hunk 2 above)
Comment (hucste) [reply content/fact/opensmtpd.md]: No pb! ;)

**Pull request merged**